templates/js-crawlee-cheerio/README.md

```diff
@@ -1,12 +1,19 @@
-# CheerioCrawler Actor template
+# JavaScript Cheerio Actor template
```

Comment:
I'd probably keep the `CheerioCrawler Actor template` as this template is not purely JavaScript + Cheerio library. It is using the CheerioCralwer which is defined by the Crawlee library (see [this](https://crawlee.dev/api/cheerio-crawler/class/CheerioCrawler) for more info)

I'd maybe go for something like `Crawlee CheerioCrawler Actor template` to make it even more clear

Reply:
I'm gonna let there language TypeScript/JavaScript to avoid confusion between templates.